assets/queries/k8s/tiller_service_not_deleted/query.rego

```diff
@@ -0,0 +1,54 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    keyword := "tiller"
+
+    metadata := document.metadata
+    contains(metadata.name,keyword)
+
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("metadata.name=%s",[metadata.name]),
+                "issueType":		"IncorrectValue",
+                "keyExpectedValue": "metadata.name does not contain 'tiller'",
+                "keyActualValue":  "document[%d].metadata.name contains 'tiller'"
+              }
+}
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    keyword := "tiller"
+
+    metadata := document.metadata
+    labels:= metadata.labels
+    some j
+        contains(labels[j],keyword)
+    
+	result := {
+                "documentId": 		input.document[i].id,
+                "searchKey": 	    sprintf("metadata.labels.%s", [j]),
```

Comment:
@joelsou5a In this case, I wasn't able to search with the name in the metadata, because the key is at the same level

Reply:
I think you should then search it on metadata itself (metadata name) otherwise we could have problems with t

"searchKey": 	    sprintf("metadata.name=%s", [metadata.name]),

its not precise but we will not have problems with mismatch of the lines.

Reply:
attended requested changes